### PR TITLE
Add font ascent and descent support

### LINKED ISSUE #16: Need FONT_ASCENT and FONT_DESCENT parameters
The latest `BDF` code in CircuitPython expects the `FONT_ASCENT` and `FONT_DESCENT` properties. See https://github.com/adafruit/Adafruit_CircuitPython_Bitmap_Font/commit/6ae6e074849738d7f344b99a7830176a5610bdb3

Also, the conversion tool https://adafruit.github.io/web-bdftopcf/ fails to convert our files without these properties.

### PR DESCRIPTION
**Pull request by bcr**
This adds the output of the `FONT_ASCENT` and `FONT_DESCENT`properties in BDF files. Fixes #16

### CODE REVIEW

**Review by FoamyGuy — APPROVED** [review]
I tested this out this morning, and the resulting fonts are working well.

Changes look good to me. Thank you!

File: blinka/commands/font.py [hunk 1]
```diff
@@ -55,22 +55,37 @@ def make_suggested_filename(font, size):
...
 
 def update_bounding(bounding_box, glyph):
     if not bounding_box:
-        bounding_box = [0, 0, 0, 0]
+        bounding_box = [0, 0, 0, 0, 0, 0]
     x = glyph.bitmap.width
     y = glyph.bitmap.height
     x_offset = 0
     y_offset = -glyph.descent if glyph.descent else glyph.ascent - glyph.height
+    ascent = glyph.ascent
```
Comment (dcaponi) [inline blinka/commands/font.py:81]: why not move this declaration up so you can use it in your y_offset calculation?

Comment (bcr) [reply blinka/commands/font.py:81]: Ignorance. But in fairness it made the diffs cleaner.

Comment (dcaponi) [reply blinka/commands/font.py:81]: makes sense. shippit

**Pull request merged**